tests/test_meshtastic_utils_message_paths.py

```
@@ -481,3 +555,235 @@ def test_on_meshtastic_message_non_text_plugin_exception(reset_meshtastic_global
+    with _patch_message_deps(plugins=[], patch_logger=False) as (
+        _mock_logger,
+        mock_relay,
+    ):
+        on_meshtastic_message(packet, _make_interface())
+
+    assert mock_relay is not None
+    mock_relay.assert_awaited_once()
+
+
+def test_on_meshtastic_message_chat_portnums_override_promotes_via_numeric_config(
+    reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)